Engine/ResourceManagement/ResourcesDB/CoreResources.h

```diff
@@ -0,0 +1,51 @@
+#ifndef _CORERESOURCES_H_
+#define _CORERESOURCES_H_
+
+#include <unordered_map>
+
+enum class CoreResource
+{
+	UNKNOWN = 0,
+	CUBE = 1,
+	SPHERE = 2,
+	TORUS = 3,
+	CYLINDER = 4,
+	QUAD = 5,
+
+	DEFAULT_MATERIAL = 6,
+
+	DEFAULT_SKYBOX = 7,
+	SKYBOX_TEXTURE_RIGHT = 8,
+	SKYBOX_TEXTURE_LEFT = 9,
+	SKYBOX_TEXTURE_UP = 10,
+	SKYBOX_TEXTURE_DOWN = 11,
+	SKYBOX_TEXTURE_FRONT = 12,
+	SKYBOX_TEXTURE_BACK = 13,
+
+	BILLBOARD_CAMERA_TEXTURE = 14,
+	BILLBOARD_LIGHT_TEXTURE = 15
```

Comment:
it's mandatory to have the equivalent (enum member = number) since enum class can do it for you?

Reply:
Not really, I made this explicit assignation to ensure that CoreModels have this specific ids. This could be useful if we add/delete some core resources. Other way, we would have to change all metafiles for core models